deploy/managed-upgrade-operator-config/10-managed-upgrade-operator-configmap.yaml

```diff
@@ -8,7 +8,7 @@ data:
     upgradeType: OSD
     configManager:
       source: OCM
-      ocmBaseUrl: ${OCM_BASE_URL}
+      ocmBaseUrl: http://ocm-agent.openshift-ocm-agent-operator.svc.cluster.local:8081
```

Comment:
fyi @bmeng MC's have their own MUO config already (https://github.com/hectorakemp/managed-cluster-config/blob/4fb9e4af8205edb8446655d96d0abbaaf129b551/deploy/managed-upgrade-operator-config/config.yaml) and (https://github.com/hectorakemp/managed-cluster-config/tree/4fb9e4af8205edb8446655d96d0abbaaf129b551/deploy/managed-upgrade-operator-config/hypershift-mc/)